### DISCUSSION TITLE
Comfy UI logo

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [comfyanonymous/ComfyUI#910] [Ideas]
Some propositions (I will keep this updated)
1.
![sComfy-UI-001](https://github.com/comfyanonymous/ComfyUI/assets/75789136/a9445b86-a979-4467-be0b-750800e655a0)
2
![Compfy UI Logo 002](https://github.com/comfyanonymous/ComfyUI/assets/75789136/ef158f5a-9dd1-4815-90c8-b098b84c62f6)
3.
![Compfy UI Logo 003](https://github.com/comfyanonymous/ComfyUI/assets/75789136/dd756111-8807-42f8-afa6-136c62a85304)
4.
![Compfy UI Logo 004](https://github.com/comfyanonymous/ComfyUI/assets/75789136/22b2f0d7-284c-4268-80e1-f0ba90f617ea)
5.
![Compfy UI Logo 005](https://github.com/comfyanonymous/ComfyUI/assets/75789136/82f5d745-c085-4ff6-bf48-a98e029fb88b)
6.
![Compfy UI Logo 006](https://github.com/comfyanonymous/ComfyUI/assets/75789136/a7219f15-0915-4d11-b967-b5e348a20f61)
7. 
![Compfy UI Logo](https://github.com/comfyanonymous/ComfyUI/assets/75789136/daf50cc1-35e3-460c-87ab-023e73160f8f)

**Comment by pto2k** [comment]
I like the thrid one which presents a node connection.

**Comment by alamonelf** [comment]
Oh hell ya! ComfyLove. Here's my vote for them on a black background. With the logo at 5-15% transparency For my workspace background. 😁

**Comment by lord-lethris** [comment]
Who needs photoshop to do Logo's, when you have ComfyUI..?
(I Like Logo 3 BTW))

![image](https://github.com/comfyanonymous/ComfyUI/assets/1736720/31de5424-5c0f-4b0c-9ee3-f42d2a17e2f1)

![ComfyUI_temp_yjych_00045_](https://github.com/comfyanonymous/ComfyUI/assets/1736720/a9296826-8841-4c29-b4d1-5eea86a3552c)

![ComfyUI_temp_vaoxc_00086_](https://github.com/comfyanonymous/ComfyUI/assets/1736720/9673a556-0491-464d-a036-83b5ba64c0b0)

[ComfyUI-logo-AI.zip](https://github.com/comfyanonymous/ComfyUI/files/12208434/ComfyUI-logo-AI.zip)

**Comment by crazyrobinhood** [comment]
> Some propositions (I will keep this updated) 1. ![sComfy-UI-001](https://private-user-images.githubusercontent.com/75789136/253827723-a9445b86-a979-4467-be0b-750800e655a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTM4Mjc3MjMtYTk0NDViODYtYTk3OS00NDY3LWJlMGItNzUwODAwZTY1NWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlNDIwNjZiMDEwMmQ4ODczNTgwYzA4NmUyYjEzN2E1NDYzYTZhNjk2NTNkNzdlMmMxMjQ0N2M3ZWE3YzVhMDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MnYIZ_9kJ1lDluVCwWZAjtXNZP-Y--8v0myuu8Hj20k) 2 ![Compfy UI Logo 002](https://private-user-images.githubusercontent.com/75789136/254317719-ef158f5a-9dd1-4815-90c8-b098b84c62f6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc3MTktZWYxNThmNWEtOWRkMS00ODE1LTkwYzgtYjA5OGI4NGM2MmY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwNjFmODVlZGExMWY2NzMxZWUzOGE5YTJjZjFlMTY5MGI5ZDkxMzI5ZmFiYmI1Mjk3NWE0YjY0YjQ1NDhjYjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uxQyvmixV0LGXuHxV3gBUZD2DxJkPKMD8N9wRgWc19E) 3. ![Compfy UI Logo 003](https://private-user-images.githubusercontent.com/75789136/254317814-dd756111-8807-42f8-afa6-136c62a85304.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc4MTQtZGQ3NTYxMTEtODgwNy00MmY4LWFmYTYtMTM2YzYyYTg1MzA0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ3M2UxZWI1Yjg4NjE2OThjMjczNWM0NzEyZjdjZDAxMThjMzg1MGMxZjgwNjkyNjRlNTg1YTQyN2E3YmE4ODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oQK51fwuoEe4z-8q2oQ-vbtVlt2zmSRJNF73S43SV8Q) 4. ![Compfy UI Logo 004](https://private-user-images.githubusercontent.com/75789136/254317845-22b2f0d7-284c-4268-80e1-f0ba90f617ea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc4NDUtMjJiMmYwZDctMjg0Yy00MjY4LTgwZTEtZjBiYTkwZjYxN2VhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxYjkzZGJhYWNlMTRjMjg1Y2RlN2Y0MWUzNTcwZWQ1YjVjMTk3OTk1OTM0NmM2ZDdjZDRhOTk5ZDUwOWIwYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jXbPJyZn6LDlJQ4akrlNwO1uYPvLwCmhXTIs1HvgS_Q) 5. ![Compfy UI Logo 005](https://private-user-images.githubusercontent.com/75789136/254317872-82f5d745-c085-4ff6-bf48-a98e029fb88b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc4NzItODJmNWQ3NDUtYzA4NS00ZmY2LWJmNDgtYTk4ZTAyOWZiODhiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjOGZhYjAwYzczZWFiMWM4Njk1MWRmMGU0M2JkOTA0ZjEyYmNhN2I2NzYwNDdlYzg1NmYwZDQ5YjlhN2I3YmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._SFJHL2a8cJuP170y3u5dToY4jugK-5X0b8g5dvOX28) 6. ![Compfy UI Logo 006](https://private-user-images.githubusercontent.com/75789136/254317897-a7219f15-0915-4d11-b967-b5e348a20f61.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc4OTctYTcyMTlmMTUtMDkxNS00ZDExLWI5NjctYjVlMzQ4YTIwZjYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2NDY1MTg5OTYzYWExNjVkMDJjM2Q4MmUxMTBjNzJiNWM3ZmI1MDhjMjFmZWQ1NjE3YmYyN2FkYjFkOGJmYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pszKvV6AWjFwn_JDXtSZ-L9oytNAaMWVY7Ni9AVetDE) 7. ![Compfy UI Logo](https://private-user-images.githubusercontent.com/75789136/254317918-daf50cc1-35e3-460c-87ab-023e73160f8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NTM2MzIsIm5iZiI6MTcxNjc1MzMzMiwicGF0aCI6Ii83NTc4OTEzNi8yNTQzMTc5MTgtZGFmNTBjYzEtMzVlMy00NjBjLTg3YWItMDIzZTczMTYwZjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDE5NTUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxNDgyNDAzNDE4OTc2NmVkZWVkZjYxMjA0ZGE5MmI1NDEyZDU3N2E2NmE0Mjk5NjdmMGVjNmJjNTU4ZDY1YTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RIoA9muG-PkC89zI1kGeoPy0DKOTipSgNmeC_JJS2Ck)

The n.3 is nice But I prefer plain logo without lettering. As an icon. Do you have the vectors?

**Comment by JLopez63** [comment]
![logoCUI](https://github.com/comfyanonymous/ComfyUI/assets/128550406/9d267caa-bb56-4b7a-98f5-bd3f22228343)

**Reply by TwoPintOhh** [reply]
I think it should include the noodles

**Reply by xtri98** [reply]
old man fashsion style. Not suitable for 2024 bro

**Comment by aimpowerment** [comment]
A nice and clean logo version for comfyui
![ComfyUI_Logo](https://github.com/user-attachments/assets/1a5a3a2c-066e-4c07-a6d5-8ef56fa3a099)
https://commons.m.wikimedia.org/wiki/File:ComfyUI_Logo.png

Alberto

**Comment by aosnec** [comment]
You may think differently. I have more than 20 years of experience in design. I see ComfyUI as a beautiful, caring girl assistant, so I am sure that the logo should convey this. Here is a sketch that conveys all the cheerfulness and care of the ComfyUI program, and also gives a positive mood and inspires creativity. The inscription chose purple for a reason. All girls love this color.
![ComfyUI](https://github.com/user-attachments/assets/e9402939-670d-4512-b179-54c27772a068)

**Reply by aosnec** [reply]
here is the full list of options.
![ComfyUI dark](https://github.com/user-attachments/assets/35c0cafd-51fe-40b8-82b4-cb79cc1d3d5d)
![ComfyUI_alpha](https://github.com/user-attachments/assets/3e6e02f6-1026-4e7a-a066-a3a6b3589bef)

**Reply by aosnec** [reply]
This is how it looks on my desktop
![ddd2](https://github.com/user-attachments/assets/c4947912-00f0-431d-9d6d-56cb0fbc747d)
